wordpress-mobile/WordPress-Android
# Fixed crash when trying to reupload nonexistent media.

Fixes #10609 

When removing an image that failed to upload, we were not removing it from the failed media list, which we use to reupload all media.

To test:
- Turn on airplane mode.
- Start a new post with Block editor.
- Add an image block and add an image from the device.
- Add another image block and add another image from the device.
- Tap on the last image and remove it.
- Tap on the first image and select "RETRY ALL" from the dialog.
- Notice that the app is not crashing.


PR submission checklist:
